nodejs/trivialOdbc/main.js

```diff
@@ -0,0 +1,14 @@
+const odbc = require('odbc');
+
+async function test() {
+  try {
+    const connection = await odbc.connect('DSN=LUGDEMO;DRIVER=IBM i Access ODBC Driver');

```

Comment:
`DRIVER=IBM i Access ODBC Driver` option can be moved into the `.odbcini` file right? That would make the connect line cleaner.